src/lib/mailer/index.js

```diff
@@ -132,7 +186,7 @@ export const notifyApproverAssigned = (job, transport = defaultTransport) => {
       displayId,
     } = report;
     const approverEmail = newApprover.User.email;
-    logger.info(`MAILER: Notifying ${approverEmail} that they were requested to approve report ${displayId}`);
+    logger.debug(`MAILER: Notifying ${approverEmail} that they were requested to approve report ${displayId}`);
```

Comment:
Should this be debug only?

Reply:
Changed it to debug since it was exposing user's email address.